app/views/articles/tag_index.html.erb

```diff
@@ -4,31 +4,34 @@
 <% expiry_minutes = params[:timeframe].blank? || params[:timeframe] == "latest" ? 4 : 20 %>
 <% flag = true %>
 <% cache("tag-stories-index-#{params}-#{flag}-#{@tag_model.updated_at}-#{user_signed_in?}", expires_in: expiry_minutes.minutes) do %>
-  <style>
-    .tag-header {
-      border: 2px solid<%= HexComparer.new([@tag_model.bg_color_hex || "#0000000", @tag_model.text_color_hex || "#ffffff"]).brightness(0.88) %>;
-      box-shadow: 5px 6px 0px<%= HexComparer.new([@tag_model.bg_color_hex || "#0000000", @tag_model.text_color_hex || "#ffffff"]).brightness(0.88) %>;
-    }
-  </style>
-  <div class="user-profile-header tag-header">
-    <div class="tag-or-query-header-container">
-      <h1>
-        <% if @tag_model.badge_id && @tag_model.badge %>
-            <img src="<%= optimized_image_url(@tag_model.badge.badge_image_url, width: 180) %>" style="transform: rotate(<%= -25 + (@tag_model.id.digits.first * 5) %>deg);" />
-        <% end %>
-        <% if @tag_model && @tag_model.pretty_name.present? %>
-          <%= @tag_model.pretty_name %>
-        <% else %>
-          <%= @tag %>
-        <% end %>
+  <div class="crayons-layout">
+    <header class="tag-header crayons-card branded-4 relative" style="border-top-color: <%= HexComparer.new([@tag_model.bg_color_hex || "#0000000", @tag_model.text_color_hex || "#ffffff"]).brightness(0.88) %>">
+      <div class="flex justify-between items-center">
+        <div class="inline-flex items-center">
+          <h1 class="crayons-title">
+            <% if @tag_model && @tag_model.pretty_name.present? %>
+              <%= @tag_model.pretty_name %>
+            <% else %>
+              <span class="crayons-tag__prefix">#</span>
```

Comment:
Since this is not really part of an actual `crayons-tag` component here, I'd just go with this instead:

```suggestion
              <span class="opacity-50">#</span>
```